### DISCUSSION TITLE
How to extend theme with expo on mobile?

### DISCUSSION THREAD

**Discussion by corynorris** [nativewind/nativewind#91] [Q&A] [answered]
Hi,

I've created a basic Expo app (via `expo init`) and followed the [babel setup guide](https://tailwindcss-react-native.vercel.app/compilation/babel) and created a `tailwind.config.ts` file:
```js
/** @type {import('tailwindcss').Config} */
module.exports = {
  content: ["./src/**/*.{js,jsx,ts,tsx}"],
  theme: {
    extend: {
      flex: {
        '2': '2 2 0%'
      }
    },
  },
  plugins: [],
}
```
Included the `babel.config.js` file:
```js
module.exports = function (api) {
  api.cache(true);
  return {
    presets: ['babel-preset-expo'],
    plugins: [["tailwindcss-react-native/babel",
      { tailwindConfig: "./tailwind.config.js" }]]
  };
};

```
As well as wrapped my app in a `TailwindProvider`. Tailwind itself works on both web and emulator, but only after specifying the `tailwind.config.js` file did the custom `flex-2` property start working on web and it still does not work on the android emulator. I'm fairly new to react-native & tailwind, is there something I've missed or misunderstood / is this just a known limitation?

Any help would be appreciated.

Thanks!

**Comment by marklawlor** [comment, marked as answer]
https://reactnative.dev/docs/layout-props#flex
https://reactnative.dev/docs/flexbox

React Native's flex is different to the web, it's only a single number. Extending the theme should work, but RN is just ignoring the invalid style.

**Reply by corynorris** [reply]
Ah, my mistake thank you for your help - I adjusted the theme to flex property as per the docs and it works now. Thanks!

**Comment by danstepanov** [comment]
Closing this discussion, if this is still relevant, feel free to open a new one.
